dev_guide/application_lifecycle/promoting_applications.adoc

```diff
@@ -684,7 +684,7 @@ pipeline:
 ----
 $ oc login <source_environment>
 $ oc project <source_project>
-$ oc export dc,is,svc,route,secret,sa -l template=<application_template_name> -o yaml > export.yaml
+$ oc export dc,is,svc,route,secret,sa -l promotion-group=<application_template_name> -o yaml > export.yaml
```

Comment:
change `application_template_name` to `application_name` as well to be consistent with the instruction above